modules/kotest/src/main/kotlin/org/testcontainers/containers/StartablePerTestListener.kt

```diff
@@ -0,0 +1,30 @@
+package org.testcontainers.containers
+
+import io.kotest.core.listeners.TestListener
+import io.kotest.core.test.TestCase
+import io.kotest.core.test.TestResult
+import org.testcontainers.lifecycle.Startable
+
+
+/**
+ * [StartablePerTestListener] starts the given [startables] before execution of each test in the spec
+ * and stops after execution of each test.
+ *
+ * [startables] can any of [GenericContainer] [ContainerisedDockerCompose] [LocalStackContainer] etc.
```

Comment:
`ContainerisedDockerCompose` here should be `DockerComposeContainer` instead

Reply:
Done